scripts/sanitycheck

```diff
@@ -689,6 +689,11 @@ class DeviceHandler(Handler):
             if options.west_flash != []:
                 command.append('--')
                 command.extend(options.west_flash.split(','))
+            # This argument is used for boards like up_squared
+            # to run sanitycheck script.
+            if options.tftp_config is not None:
```

Comment:
This does not belong into sanitycheck, all of this needs to be supported in west and the flasher and provided to sanitycheck as an option via --west-flash

Reply:
Do you mean that I don't need to add another option, I should just use the existing --west-flash option?
for now, I can use west flash --option well by adding support to tftp runner. yes, I can just add the option behind --west-flash, don't need to modify sanitycheck for other options.